### PR TITLE
Project model

### PR DESCRIPTION
**Pull request by d-ang1993**

### CODE REVIEW

File: server/migrations/20190822164125-create-project.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      photos: {
+        type: Sequelize.ARRAY(Sequelize.STRING)
+        // allowNull: false,
+        // unique: true
```
Comment (skassam21) [inline server/migrations/20190822164125-create-project.js]: You probably want to remove these comments, not necessary

File: server/migrations/20190822164125-create-project.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        type: Sequelize.STRING
+      },
+      link: {
+        type: Sequelize.STRING
+      },
+      userId: {
+        type: Sequelize.INTEGER,
+        onDelete: "CASCADE",
+        allowNull: false
+        // references: { model: "Users", key: "id" }
```
Comment (skassam21) [inline server/migrations/20190822164125-create-project.js]: why not keep the reference here?

File: server/routes/users.js [hunk 3]
```diff
@@ -1,11 +1,33 @@
...
+    console.log(err);
+  }
+});
+
+router.get("/all", function(req, res, next) {
+  try {
+    User.findAll({
+      include: [Project]
+    }).then(users => {
+      console.log(users[0].Projects);
```
Comment (skassam21) [inline server/routes/users.js:30]: These routes are just used for testing right and will be modified in the future? Looks like you have the right relationships that we need

Comment (d-ang1993) [reply server/routes/users.js:30]: Yup these routes are just for testing, and will be removed in the future. Okay great :] thanks for the review!

**Review by wamts — APPROVED** [review]
Looks good!

**Review by wamts — APPROVED** [review]

**Review by kidistz — APPROVED** [review]
Looks good.

**Pull request merged**